js/protoblocks.js

```diff
@@ -1357,6 +1638,9 @@ class ProtoBlock {
     }
 
     // E.g., color, shade, pensize, ...
```

Comment:
move

Reply:
so sorry for the inconvenience caused. I'll look into it.